fe/fe-core/src/main/java/org/apache/doris/nereids/NereidsPlanner.java

```diff
@@ -71,6 +93,9 @@ public class NereidsPlanner extends Planner {
 
     public NereidsPlanner(StatementContext statementContext) {
         this.statementContext = statementContext;
+        if (ConnectContext.get().getSessionVariable().isEnableNereidsTrace()) {
+            EventChannel.getDefaultChannel().start();
+        }
```

Comment:
each statement has a brand new NereidsPlanner. So why start channel here? we need to start channel for each statement?